frontend/src/components/Profile/Profile.tsx

```diff
@@ -9,19 +9,35 @@ import {
 } from '@mui/material';
 import { AccountCircle } from '@mui/icons-material';
 import ComponentContainer from '../common/ComponentContainer';
+import useUser from '@/hooks/useUser';
+import { useClient } from '@/contexts/AppContext';
 
 const InnerProfile: React.FC = () => {
-  const [formData, setFormData] = useState({
+  type UserMetadata = {
```

Comment:
Can this be put in a separate file please. Helps keep things neat and tidy (and reusable)

Reply:
Will put innerProfile into signle file. Thanks Mursal, it is a good suggestion.

Reply:
And I have tried changing and updating details, the supabase is sucessfully updated

Reply:
> Will put innerProfile into signle file. Thanks Mursal, it is a good suggestion.

Sorry meant the type `UserMetaData`.

Reply:
Now I get your point XD, my bad.

Reply:
the `userTypes` file is empty

Reply:
Now should be fine